packages/catlog/src/simulate/ode/cclfo.rs

```diff
@@ -0,0 +1,33 @@
+//! Constant-coefficient linear first-order (CCLFO) differential equations.
+
+use nalgebra::{DMatrix, DVector};
+
+#[cfg(test)]
+use super::ODEProblem;
+use super::ODESystem;
+
+/** A CCLFO dynamical system.
+
+A system of linear first-order ODEs with constant coefficient; a semantics for
+enriched causal loop diagrams.
+*/
+#[derive(Clone, Debug, PartialEq)]
+pub struct CCLFOSystem {
+    interaction_coeffs: DMatrix<f32>,
```

Comment:
Since there's only one type of coefficient/parameter in this case, I'd just call this `coeffs` or `coefficients`.